src/formdata.js

```diff
@@ -7,13 +7,12 @@ const form = document.querySelector('#form-id');
 const refresh = document.querySelector('#refresh-score-btn');
 const table = document.querySelector('table');
 
```

Comment:
- [ ] Please add a short description of your project, currently, your project has no description, this is what I see in your current README file:
    - Please add a short description, what were the goals for this project or what did you learned can look good here 😄 
    ![image](https://user-images.githubusercontent.com/60789317/141361933-8721f122-c987-43fa-bde3-140ed3ffcd0c.png)

Reply:
- [ ] On the `Setup` section of your README file, have this `- npm install webpack webpack-cli --save-dev`, install dependencies is not necessary, they are already on the `package.json`, please add something like this instead 😄:
    - `npm install`, nothing else, this will install all necessary packages 🚀 
    - I will recommend you to take a look to this [LINK](https://heynode.com/tutorial/what-packagejson/)

Reply:
- [ ] You did not change the `Acknowledgments` of your README, please add some acknowledgments or delete it 🚀

Reply:
- [ ] On your `Built With` section you said that you use `frameworks` but did not specify which ones, if you have made use of any framework please add it, otherwise, please delete this part from your README file 😄